vars/getTestResults.groovy

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import hudson.tasks.test.AbstractTestResultAction
+import groovy.json.JsonOutput
+import java.text.SimpleDateFormat
+import java.util.Date
+
+void call(Map args = [:]) {
+    def lib = library(identifier: 'jenkins@main', retriever: legacySCM(scm))
+    def finalJsonDoc = ""
+    currentBuild.description = "runner: Ubuntu2004 M58xlarge<br><a href=\"https://github.com/opensearch-project/OpenSearch/pull/12503\">PR #12503</a>: removing unused fetch sub phase processor initialization during fetch"
```

Comment:
Do we hardcode this PR number here?

Reply:
Sorry, this was purely for testing. Let me remove it. 
Also making PR draft as I am awaiting cluster setup to complete.